editor/src/components/editor/actions/actions.tsx

```diff
@@ -734,7 +734,7 @@ function switchAndUpdateFrames(
     case 'flex':
       withUpdatedLayoutSystem = {
         ...withUpdatedLayoutSystem,
-        allElementProps: MetadataUtils.setPropertyDirectlyIntoMetadata(
+        _currentAllElementProps_KILLME: MetadataUtils.setPropertyDirectlyIntoMetadata(
```

Comment:
If you don't kill this in the next few days I will give you my extreme disappointment face